contrib/kafka/config/kafka.yaml

```diff
@@ -170,6 +170,13 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - eventing.knative.dev
```

Comment:
Line numbers are confusing, so instead I'll reply to the comment's context.
"Do we want to remove the Istio injection for the Dispatcher"
With the first wave of changes I am not removing it because 
1. there are other channels in eventing that still create virtual services and there could always be a scenario that a subscriber to kafka dispatcher could be a GCP channel. (Correct me if this is not possible). So all PRs for channels are in, I'll send another PR that will remove injection.
2. I still need to explore the case where someone needs Istio or some mesh for its features. Referring to the [discussion on slack](https://knative.slack.com/archives/C9JP909F0/p1555549611098300). I'll connect with folks from serving tomorrow to understand how the optional mesh and pluggable mesh support works?